rockboxalbumfix.py

```diff
@@ -197,16 +235,20 @@ def process_images(root_dir, iteration_timeout=5, max_retries=3):
     except KeyboardInterrupt:
         print("\nProcessing interrupted by user.")
 
+
 def clear_temp_directory():
-    temp_folder = os.getenv('TEMP')
-    temp_folder_path = os.path.join(temp_folder, 'cover_extraction_temp', 'cover_extraction_temp')
+    temp_folder = os.getenv("TEMP")
```

Comment:
missed this small part, still uses os.getenv but will fix before pulling!

Reply:
Fixed!